### PR TITLE
Update community rule 1 to prevent wrangling weasels wresting worryingly wide

### PR DESCRIPTION
**Pull request by Ephemeralis**
I seriously cannot believe I had to write this.

### CODE REVIEW

File: wiki/Rules/en.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ These rules are the basis from which we help keep the osu! community a fun and w
 Community Rules
 ---------------
 
-1.  **Each player may only have *ONE* account at any point of time.** This account is you. It is not anyone else - not your brother, your mother, your sister, your friend - it is YOU. Don't share your account with anyone else.
+1.  **Each player may only have *ONE* account, ever.** The *first* account you make during account registration is the account you will have in *osu!* forever. This account is YOU. It is not anyone else - not your brother, your mother, your sister, your friend - it is YOU. Don't share your account with anyone else. You don't get to make a new one if you lose access to it. Keep it safe.
```
Comment (TPGPL) [inline wiki/Rules/en.md]: You don't get into You mustnt?

Comment (TPGPL) [inline wiki/Rules/en.md]: you will have in *osu!* forever. -> you will only use in *osu!*?

Comment (Ephemeralis) [reply wiki/Rules/en.md]: musn't implies it is a choice they can, but should not consider

'don't get' is conversational for 'you cannot do this. ever.'

Comment (Ephemeralis) [reply wiki/Rules/en.md]: this one is worded funny and I'll change it

**Pull request merged**